lib/python/qmk/keymap.py

```diff
@@ -27,40 +27,38 @@
  * edit it directly.
  */
 __KEYCODE_OUTPUT_GOES_HERE__
-
-const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 __KEYMAP_GOES_HERE__
-};
-
-#if defined(ENCODER_ENABLE) && defined(ENCODER_MAP_ENABLE)
-const uint16_t PROGMEM encoder_map[][NUM_ENCODERS][NUM_DIRECTIONS] = {
 __ENCODER_MAP_GOES_HERE__
-};
-#endif // defined(ENCODER_ENABLE) && defined(ENCODER_MAP_ENABLE)
-
 __MACRO_OUTPUT_GOES_HERE__
 
+#ifdef OTHER_KEYMAP_C
+#if __has_include_next(OTHER_KEYMAP_C)
+#include_next OTHER_KEYMAP_C
+#endif // __has_include_next(OTHER_KEYMAP_C)
+#endif // OTHER_KEYMAP_C
 """
 
 
 def _generate_keymap_table(keymap_json):
-    lines = []
+    lines = ['const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {']
```

Comment:
I think that this will require anything using existing `<keyboard>/templates/keymap.c` to be updated?

Reply:
Groan, yeah, seems like it.